_data/simple-icons.json

```
@@ -11433,6 +11433,11 @@
             "hex": "20336B",
             "source": "https://paytm.com"
         },
+        {
+            "title": "PBS",
+            "hex": "2135c5",
```

```suggestion
            "hex": "2638C4",
```
Not sure where you got this other hex from.